libvmi/driver/kvm/kvm.c

```diff
@@ -52,6 +53,9 @@
 #include <sys/time.h>
 #include "driver/kvm/include/kvmi/libkvmi.h"
 
+/*
+ * Helpers
+ */
 static uint32_t
 translate_msr_index(int index, int *err) {
```

Comment:
this function will be removed thanks to `msr-index` array:
https://github.com/KVM-VMI/libvmi/pull/9/files#diff-a342220ea96575ee149235454636ce4eR55